src/tools/lscgroup.c

```diff
@@ -47,6 +47,9 @@ static void usage(int status, const char *program_name)
 	info("  -g <controllers>:<path>	Control group to be ");
```

Comment:
nitpick - First line of the commit comment mentions `cgcreate` :)

Reply:
copy, and paste typos, I tried being cautious by still, sorry :(